.github/workflows/style.yaml

```diff
@@ -0,0 +1,53 @@
+name: Style
+on:
+  pull_request:
+    types:
+    - opened
+    - synchronize
+    - labeled
+
+env:
+  ROX_PRODUCT_BRANDING: RHACS_BRANDING
+
+jobs:
+
+  style-check:
+    if: |
+      (github.event.action == 'labeled' && github.event.label.name == 'turbo-build') ||
+      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'turbo-build'))
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.50
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Cache Go dependencies
+        uses: ./.github/actions/cache-go-dependencies
+
+      - uses: ./.github/actions/create-concatenated-ui-monorepo-lock
+
+      - name: Cache UI dependencies
+        uses: ./.github/actions/cache-ui-dependencies
+
+      - name: Cache QA Test dependencies
+        uses: ./.github/actions/cache-gradle-dependencies
+
+      - name: PR labels
+        uses: joerick/pr-labels-action@v1.0.6
```

Comment:
I would suggest not add 3-rd party dependency for the task to determine if a label is present on PR.
Some googling suggests that `${{contains( github.event.pull_request.labels.*.name, 'My Label')}}` will provide an answer if `My Label` is present.

Reply:
I think this step is not required anyway but I'll fix build.yaml

Reply:
https://github.com/stackrox/stackrox/pull/4049